libraries/ssh_crypto.rb

```diff
@@ -256,7 +256,7 @@ def valid_algorithms # rubocop:disable Metrics/CyclomaticComplexity
 
   # returns the hostkeys value based on valid_algorithms
   def valid_hostkeys
-    hostkeys = valid_algorithms.map { |alg| "#{sshd_custom_path}/ssh_host_#{alg}_key" }
+    hostkeys = valid_algorithms.map { |alg| "#{inspec.input('sshd_custom_path')}/ssh_host_#{alg}_key" }
```

Comment:
Since only one test is affected we should probably that specific handling directly to the control: https://github.com/dev-sec/ssh-baseline/blob/fef3eb4893c2ac1089e63c491e3e73632aa69b32/controls/sshd_spec.rb#L183-L190

It will also be easier for the reader of the control to understand what the control is actually doing